packages/ui/src/components/field.tsx

```
@@ -0,0 +1,236 @@
+      return children
+    }
+
+    if (!errors?.length) {
+      return null
+    }
+
+    const uniqueErrors = [
+      ...new Map(errors.map((error) => [error?.message, error])).values(),
+    ]
+
+    if (uniqueErrors?.length == 1) {
```

Biome (2.4.13)

[error] Using == may be unsafe if you are relying on type coercion.

(lint/suspicious/noDoubleEquals)